.gitignore

```diff
@@ -1,10 +1,8 @@
-# Transpiled CSS files
-
+# Showcase directory
 /showcase
 
-styles.css
-styles.min.css
-
-# Source maps
-styles.css.map 
-styles.min.css.map 
\ No newline at end of file
+# Transpiled CSS
```

Comment:
It could be better to ignore CSS files from the CSS directory (as you have described in the README) - do not ignore each CSS file globally because if you want to embed some library from GH, then it won't be visible for the git.